tests/main/test_predictions_wrapper.py

```diff
@@ -37,23 +37,35 @@ def verify_pickle_serialization(self, model, model_wrapper, test_data):
 
 
 class TestPredictionsWrapperClassification(TestPredictionsWrapper):
-    @pytest.mark.parametrize('dataset_name', ['iris', 'titanic'])
-    def test_prediction_wrapper_classification(self, iris, titanic_simple, dataset_name):
+    @pytest.mark.parametrize('dataset_name', ['iris', 'titanic', 'cancer', 'wine', 'multiclass'])
```

Comment:
nit: move some of these repeating strings to constants here

Reply:
Will leave these as strings as these are in tests.